### DISCUSSION TITLE
GatsbyImage and StaticImage Does Not Load on Deployment

### DISCUSSION THREAD

**Discussion by abechoi** [gatsbyjs/gatsby#31279] [Help]
Gatsby noob here, trying to finish my first Gatsby project but for some reason, every time I deploy my site with AWS Amplify, the images do not load! My images load using GatsbyImage and StaticImage for gatsby develop and build/serve, but every time I deploy it, it does not.

Please help, I'm about to break my computer, buy a new one, and redo my project in Vanilla React.

abechoi.com

test.abechoi.com

https://github.com/abechoi/Abechoi_Gatsby

**Comment by abechoi** [comment]
I deployed this on Netlify and it works fine, I suppose it's an AWS Amplify issue. I'll take it there... If anyone can still provide any insight, I would greatly appreciate it!

**Comment by LekoArts** [comment]
You only see the blurry placeholder as the images don't seem to be available at the URLs that the image tags are pointing to. If you haven't checked out our own guide (https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/deploying-to-aws-amplify/) or their own community forums I'd recommend doing so :) This seems like a deployment issue and unrelated to Gatsby -- hope you find your problem!

**Reply by abechoi** [reply]
I appreciate the doc! I created a service role and added it, but unfortunately, still no change. Thanks anyways.

**Reply by devrushi** [reply]
Hi @abechoi,
I developed some banners using the <StaticImage> and it worked fine but messed up on AWS Amplify after a week of searching (almost lost all hopes) I tried the formats={["AUTO"]} attribute and it fixed for me on the server.
Please give it a try.
Currently, I m stuck in <GatsbyImage> as I need algolia data to bring me an image again losing hopes 😅
